source/guides/references/error-messages.md

```diff
@@ -529,15 +529,31 @@ It's possible to enable debugging these scripts by adding the `crossorigin` attr
 
 # Browser Errors
 
-## {% fa fa-exclamation-triangle red %} The Chromium Renderer process just crashed
+<!-- keep old hash -->
+<a name='The-Chromium-Renderer-process-just-crashed'></a>
 
-Browsers are enormously complex pieces of software, and from time to time they will inconsistently crash *for no good reason*. Crashes are just a part of running automated tests.
+## {% fa fa-exclamation-triangle red %} The browser process running your tests just exited unexpectedly
 
+This error can occur whenever Cypress detects that the browser that was launched for running tests has exited or crashed before the tests could finish running, or without the user manually stopping the tests in Cypress.
+
+This can happen for a number of different reasons, including:
+
+- The browser was exited manually, by clicking the "Quit" button or otherwise
+- Your test suite or website code is starving the browser for resources, such as with an infinite loop
+- Tests are running in a memory-starved environment
+- The browser is testing a memory-heavy application
+- Tests are running within Docker (there is an easy fix for this: see {% issue 350 'this thread' %})
+- There are problems with the GPU / GPU drivers
+- There is a bug in the browser <!-- TODO: link to Firefox bug here? https://github.com/cypress-io/cypress/issues/6187-->
+- There is a memory leak in Cypress
+
+<!-- TODO: worth adding image of new error?
```

Comment:
@flotwig This comment broke the rendering of the page (likely because it has a tag in a comment which hexo doesn't handle well)- so that none of the errors below it display on the page. Please make sure to run the docs locally to double check these things.

Reply:
Believe it or not, I did run the docs locally while making these changes :smile: I didn't notice that this was happening... that's pretty lame. Sometimes it feels like hexo has more bugs than features.

Reply:
Yes, but I think a lot of it is due to our being stuck on very old versions that no one has dedicated time to update.